tools/testing/selftests/seccomp/seccomp_bpf.c

```diff
@@ -3976,8 +3980,14 @@ TEST(user_notification_addfd)
 	ASSERT_GE(pid, 0);
 
 	if (pid == 0) {
+		/* fds will be added and this value is expected */
 		if (syscall(__NR_getppid) != USER_NOTIF_MAGIC)
 			exit(1);
+
+		/* Atomic addfd+send is received here. Check it is a valid fd */
+		if (fcntl(syscall(__NR_getppid), F_GETFD) == -1)
```

Comment:
You could add a check that `syscall(__NR_getppid), F_GETFD)` returns the fd number requested in the other side. In this case, `0` (line 4074 `addfd.newfd = 0;`). I suggest to try 43, so that we test that the fd is correctly returned by the syscall.

Reply:
Not 0 in this case, that is ignored. Also as said in the other comment the setting fd func is tested. I can _also_ add a test combining them, but I won't like to mix the tests of different flags.

Reply:
I think it's good to have a test that combine the too.